pizza_cutter/des_pizza_cutter/_des_info.py

```diff
@@ -74,24 +63,6 @@ def get_des_coadd_tile_info(
             'image_flags' : any flags for the SE image
     """
 
-    coadd_srcs = desmeds.coaddsrc.CoaddSrc(
-        medsconf,
-        tilename,
-        band,
-        campaign=campaign,
-    )
-
-    coadd = desmeds.coaddinfo.Coadd(
-        medsconf,
-        tilename,
-        band,
-        campaign=campaign,
-        sources=coadd_srcs,
-    )
-
-    info = coadd.get_info()
-    info['tilename'] = tilename
-    info['band'] = band
```

Comment:
The tilename and band info is used in other parts of the code. We should either take that out or add it back.

Reply:
that info is already in the info dict, so no need to add it again